### ISSUE TITLE
View all boxes

### ISSUE THREAD

**Issue by jimishsoni1990** [hivedb/hive#504]
Can we have a something like phpMyAdmin type of admin screen where we can see all the boxes and their values.

**Comment by TheMisir** [comment]
Nope. Cause hive stores data in binary format it's not possible to display data in user readable format (Adapters are used for parsing that data).

**Comment by jimishsoni1990** [comment]
ok, thank you for the reply.

**Comment by svonidze** [comment]
Do you know maybe someone created at least a (Flutter) widget to display keys (data inside on demand)? Do not want to invent my own wheel.
